### DISCUSSION TITLE
Refund request

### DISCUSSION THREAD

**Discussion by holtmeister** [twentyhq/twenty#2531] [General]
I signed up for what I thought was a demo of this product. I have since been charged with my credit card. There is no way of contacting anyone, and there was no receipt sent. There is also no way to cancel my account. I have sent an email to contact@twenty.com

Please cancel and refund this charge, otherwise I will be forced to dispute it with my bank.

**Comment by holtmeister** [comment]
Nobody has responded to this message, or my email, so I will proceed with filing a dispute with my bank.

**Reply by FelixMalfait** [reply]
Ouuuuch so sorry we didn't see this before. We rarely check the conversations tab only the issues tab. 
We obviously accepted the charge dispute. Apologies!
FYI there is a support chat in the app (bottom left) where we usually handle this kind of request.
